src/components/app/MenuButtons/Publish.css

```diff
@@ -175,3 +175,10 @@
 .menuButtonsPublishError {
   margin: 10px 0;
 }
+
+/* Do not animate the publish animation for stripes at the top loading bar. */
+@media (prefers-reduced-motion) {
+  .menuButtonsPublishUploadBarInner {
+    animation: none;
```

Comment:
Optional: in general I think I'd rather have `@media (prefers-reduced-motion: no-preference)` that adds the animation instead of  ` @media (prefers-reduced-motion) {` removing it. I think it makes it easier to maintain in the longer term, reducing the risk of typos.